scripts/apps/search/MultiImageEdit.ts

```diff
@@ -44,7 +44,7 @@ export function MultiImageEditController(
         const updatedImages = imagesOriginal.map((image) => {
             const existingImage = $scope.images.find(({_id}) => _id === image._id);
 
-            return existingImage != null ? existingImage : {...image, ...$scope.metadata};
+            return existingImage != null ? existingImage : {...image, ...pickBy($scope.metadata)};
```

Comment:
Are you picking everything? Do you need to pick at all then?

Reply:
I'm picking everything that's not empty

Reply:
Can you then use `(item) => item != null` predicate. It's sneaky that lodash is silently ignoring `null`.

Reply:
added explicit check